src/otx/algo/classification/efficientnet.py

```diff
@@ -25,41 +24,36 @@
 from otx.algo.classification.necks.gap import GlobalAveragePooling
 from otx.algo.classification.utils import get_classification_layers
 from otx.algo.utils.support_otx_v1 import OTXv1Helper
-from otx.core.data.entity.base import OTXBatchLossEntity
-from otx.core.data.entity.classification import (
-    HlabelClsBatchDataEntity,
-    HlabelClsBatchPredEntity,
-    MulticlassClsBatchDataEntity,
-    MulticlassClsBatchPredEntity,
-    MultilabelClsBatchDataEntity,
-    MultilabelClsBatchPredEntity,
-)
-from otx.core.exporter.base import OTXModelExporter
-from otx.core.exporter.native import OTXNativeModelExporter
-from otx.core.metrics import MetricInput
-from otx.core.metrics.accuracy import HLabelClsMetricCallble, MultiClassClsMetricCallable, MultiLabelClsMetricCallable
+from otx.core.metrics.accuracy import DefaultClsMetricCallable
 from otx.core.model.base import DefaultOptimizerCallable, DefaultSchedulerCallable
-from otx.core.model.classification import OTXHlabelClsModel, OTXMulticlassClsModel, OTXMultilabelClsModel
+from otx.core.model.classification import OTXClassificationModel
 from otx.core.schedulers import LRSchedulerListCallable
 from otx.core.types.label import HLabelInfo, LabelInfoTypes
+from otx.core.types.task import OTXTaskType, OTXTrainType
 
 if TYPE_CHECKING:
     from lightning.pytorch.cli import LRSchedulerCallable, OptimizerCallable
 
-    from otx.core.metrics import MetricCallable
+    from otx.core.metrics import MetricCallablePerTask
 
 
-class EfficientNetForMulticlassCls(OTXMulticlassClsModel):
-    """EfficientNet Model for multi-class classification task."""
+class EfficientNetForClassification(OTXClassificationModel):
```

Comment:
I think `ForClassification` can be ignored because it is already located in `algo/classification`.
What do you think?

Reply:
I also agree with that.

Reply:
I agree to a certain extent. It's hard to tell the tasks apart by looking at the class name, but the import address does, right?
@kprokofi What do you think about this, it's a simple naming conundrum, but in the future we may have a model that supports multiple tasks simultaneously, what should it be called?

Reply:
First, I'm going to clear the `ForClassification`.